Tribler/Test/Core/test_launch_many_cores.py

```diff
@@ -176,3 +177,69 @@ def test_load_communities(self):
         self.assertTrue(self.get_community(GigaChannelCommunity))
         self.assertTrue(self.get_community(MarketCommunity))
         self.assertTrue(self.get_community(TrustChainCommunity))
+
+
+class TestLaunchManyCoreSeederBootstrapSession(TestAsServer):
+
+    def setUpPreSession(self):
+        TestAsServer.setUpPreSession(self)
+
+        # Enable all communities
+        config_sections = ['trustchain', 'ipv8', 'bootstrap', 'libtorrent']
```

Comment:
If I'm not mistaken, this test is connecting to the real TrustChain network and IPv8 bootstrap servers. Since this can lead to unexpected behaviour, we should avoid this. Why do you need trustchain/IPv8 enabled here and can you avoid this?

Reply:
Bootstrap module will be not reachable if trustchain and ipv8 are not available, since it depends on direct payouts, creating blocks etc.

Reply:
But I agree that we don't need them in this test

Reply:
Hmm this is an interesting one. I assume that if you leave out this test, it results in much uncovered code in `LaunchManyCore`. It is a bit hard to test since it's part of the `init` method when starting Tribler.

Still, having IPv8/TrustChain is undesired. Even if you replace IPv8 immediately after startup, it could be enough for IPv8 to do a bootstrap attempt and sending introduction requests to the bootstrap servers.